### DISCUSSION TITLE
3.2

### DISCUSSION THREAD

**Discussion by NattyNarwhal** [SubmarinerApp/Submariner#213] [Announcements]
Note that this release contains larger database schema updates. It's recommended you back up your Submariner directory (in your Music directory) before updating.

*Changes since 3.1.1*:

* Inspector window has been improved
  * You can now switch between now playing and current selection with a tab view.
  * Playlist information can be viewed and edited.
* The playlist model has been corrected to handle multiple of the same track. (GH-192)
* Added a command to play the first disc from an album. (GH-211)
* Lower case artist names are properly sorted in the artist list.
* Favouriting items is available from the menu bar, or Cmd+E.
* Adding items to the tracklist now has a shortcut of Cmd+D.
* You can seek to a specific timestamp. (GH-84)
* You can now adjust the speed of playback. (GH-83)
* Fix parsing fractional dates returned by some servers.
* Fix the null cover being used for system now playing information.
* Upgrading from Submariner 1.x is no longer supported.

<hr /><em>This discussion was created from the release <a href='https://github.com/SubmarinerApp/Submariner/releases/tag/v3.2'>3.2</a>.</em>

**Comment by bbsixzz** [comment]
Thank you, compiled binary on the way?

**Reply by NattyNarwhal** [reply]
Oops, I forgot to attach it. Here: https://github.com/SubmarinerApp/Submariner/releases/download/v3.2/Submariner-3.2.zip

MAS is updating as well.
